.gitignore

```diff
@@ -3,8 +3,7 @@
 .idea/
 out/
 target/
-java/
```

Comment:
generated javadoc files typically show up in `java/{version}/`. Will making this `./java` fix it?